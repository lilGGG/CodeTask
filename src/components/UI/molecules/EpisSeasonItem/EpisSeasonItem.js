import  Image  from '../../atoms/Image/Image';
import TitleEpisSeason from '../../atoms/TitleEpisSeason/TitleEpisSeason';

function EpisSeasonItem({
  id, number,itemName,srcImg,img,url
}) {
  //  console.log("url", `${itemName} ${id} ${url}` );
   const unique = number;
  //  console.log( unique, "uniq");
  return (
    <div> 
          {/* <li> {img? <a href={url} target="_blank">   <Image src={img.medium}  />  </a>: ""} </li> */}
          <li> {img? <a href={url} target="_blank">   <Image SrcImage={srcImg}  />   </a>: ""} </li> 
          {itemName? <li>  <TitleEpisSeason Text ={itemName}>  </TitleEpisSeason>  </li>: "" }
    </div>
  );
}

export default EpisSeasonItem;
