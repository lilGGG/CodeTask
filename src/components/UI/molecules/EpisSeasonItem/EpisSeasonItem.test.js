import React from "react";
import EpisSeasonItem from "./EpisSeasonItem";
import ReactDom from "react-dom";
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

describe('EpisSeasonItem test', () => {
    it('renders snapshot', () => {
      const tree = renderer.create(<EpisSeasonItem id="1" number="1"
      itemName="Epis name" srcImg="https://static.tvmaze.com/uploads/images/medium_portrait/45/113621.jpg" img={true} url="https://www.tvmaze.com/seasons/7/bitten-season-1"/>).toJSON();
      expect(tree).toMatchSnapshot();
    });

  });
  
