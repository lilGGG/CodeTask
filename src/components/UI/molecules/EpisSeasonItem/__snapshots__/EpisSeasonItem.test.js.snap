// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EpisSeasonItem test renders snapshot 1`] = `
<div>
  <li>
     
    <a
      href="https://www.tvmaze.com/seasons/7/bitten-season-1"
      target="_blank"
    >
         
      <img
        className="sc-bczRLJ fASZzU"
        src="https://static.tvmaze.com/uploads/images/medium_portrait/45/113621.jpg"
      />
         
    </a>
     
  </li>
  <li>
      
    <h1
      className="sc-gsnTZi bcJCuA"
    >
      Epis name
    </h1>
      
  </li>
</div>
`;
